lib/tasks/rummager.rake

```diff
@@ -0,0 +1,31 @@
+namespace :rummager do
+  desc "index all published documents in rummager"
+  task index: :environment do
+    require 'plek'
+    require 'rummageable'
+
+    content = Guide.all.map do |guide|
+      edition = guide.latest_edition
+      if edition.published?
+        puts "Indexing #{edition.title}..."
+        {
+          "_type": "manual_section",
+          "description": edition.description,
+          "indexable_content": edition.body,
+          "title": edition.title,
+          "link": guide.slug,
+          "manual": "service-manual",
```

Comment:
This field should have a leading slash (ie `/service-manual`).  It is expected to contain the base path of the manual.  Eg, existing values we have are "/guidance/digital-and-technology-skills".